test/extended/networking/network_segmentation.go

```diff
@@ -37,6 +37,9 @@ import (
 )
 
 const openDefaultPortsAnnotation = "k8s.ovn.org/open-default-ports"
+const totalPollTimeout = 2 * time.Minute
+const periodicPollTimeInterval = 6 * time.Second
```

Comment:
The extra words make me a bit confused, maybe `pollTimeout` and `pollInterval`?

Reply:
yea I overcomplicated the names, let me fix this